services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/_core/_events_utils.py

```diff
@@ -132,7 +132,17 @@ async def service_free_reserved_disk_space(
     app: FastAPI, node_id: NodeID, sidecars_client: SidecarsClient
 ) -> None:
     scheduler_data: SchedulerData = _get_scheduler_data(app, node_id)
-    await sidecars_client.free_reserved_disk_space(scheduler_data.endpoint)
+    try:
+        await sidecars_client.free_reserved_disk_space(scheduler_data.endpoint)
+    except BaseHttpClientError as e:
+        _logger.warning(
+            (
+                "Could not free reserved disk space for "
+                "%s\n%s. Will continue to save the data from the service!"
+            ),
+            scheduler_data.service_name,
+            f"{e}",
```

Comment:
can we make the original exception for prominent in the message?

exception `__str__` (e.g. `f"{type(e)}: {e}"`)